packaging/helm/skipper-aws/templates/kube2iam/daemonset.yaml

```diff
@@ -0,0 +1,67 @@
+{{ if .Values.install_kube2iam }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube2iam
+  namespace: kube-system
+  labels:
+    application: kube2iam
+    version: {{ .Values.kube2iamVersion }}
+spec:
+  updateStrategy:
+    type: OnDelete
+  selector:
+    matchLabels:
+      application: kube2iam
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        application: kube2iam
+        version: {{ .Values.kube2iamVersion }}
+    spec:
+      dnsConfig:
+        options:
+          - name: ndots
+            value: "1"
+      priorityClassName: system-node-critical
```

Comment:
If you change the namespace to be configurable you'll need some check here to see if it's set to `kube-system` or not.

Reply:
We could pass an if `namespace == kube-system` apply this line

Reply:
yeah that's pretty much what I'm thinking.

Reply:
fixed